doc/nrf/libraries/networking/nrf_cloud_pgps.rst

```diff
@@ -24,7 +24,7 @@ These ephemeris predictions are downloaded from the cloud, stored by the device
 
    If two-week prediction sets are used, TTFF towards the end of the second week will increase due to accumulated errors in the predictions and decrease in number of satellite ephemerides in the later prediction periods.
 
-If nRF Cloud services such as P-GPS or A-GPS are used either individually or in combination, the broadcasted information and future estimates of position can be downloaded at a faster rate from nRF Cloud than from the GPS satellites.
+If nRF Cloud services such as P-GPS or A-GPS are used either individually or in combination, the broadcasted information and future estimates of satellite positions can be downloaded at a faster rate from nRF Cloud than from the GPS satellites.
```

Comment:
Instead of "satellite position", should we talk about "satellite orbit" or "satellite orbital data"? The data doesn't contain the position of the satellite, but it can be used to calculate the position of the satellite at a specific time.

Reply:
"the broadcasted information and estimates of satellite data" is ok with me, but I still prefer "the broadcasted information and estimates of satellite orbital data"

Reply:
Not sure if my understanding is correct, but I think "broadcasted information" refers to A-GPS and "future estimates" to P-GPS. Maybe we should keep the word "future" as well, to make the separation between A-GPS and P-GPS more distinct. And maybe P-GPS and A-GPS in the first sentence should be swapped? And what about replacing "estimate" with "prediction"? 😄 

```suggestion
If nRF Cloud services such as A-GPS or P-GPS are used either individually or in combination, the broadcasted information and future predictions of satellite data can be downloaded at a faster rate from nRF Cloud than from the GPS satellites.
```